node/bft/src/bft.rs

```diff
@@ -692,7 +692,7 @@ impl<N: Network> BFT<N> {
                     // Initialize a callback sender and receiver.
                     let (callback_sender, callback_receiver) = oneshot::channel();
                     // Send the subdag and transmissions to consensus.
-                    consensus_sender.tx_consensus_subdag.send((subdag, transmissions, callback_sender)).await?;
+                    consensus_sender.tx_consensus_subdag.send((subdag.clone(), transmissions, callback_sender)).await?;
```

Comment:
hmm, cloning the `Subdag` may get expensive; could we apply the telemetry on the receiving end of the channel that's used here?

Reply:
We could, just need to extend the feature flag to the `Consensus` crate.

I also noticed that we are cloning the `commit_subdag` above as well.